rocAL/rocAL/include/loaders/circular_buffer.h

```diff
@@ -37,6 +37,11 @@ struct decoded_image_info
     std::vector<uint32_t> _roi_height;
     std::vector<uint32_t> _original_width;
     std::vector<uint32_t> _original_height;
+    std::vector<uint32_t> _roi_audio_samples;
```

Comment:
Can we not have another structure called decoded_audio_info?

Reply:
This is somehow not working